services/fuzzing-decision/src/fuzzing_decision/decision/providers.py

```diff
@@ -73,7 +73,7 @@ def get_worker_config(
                 }
             )
             if worker_type == "d2g":
-                out["genericWorker"]["config"]["enableD2G"] = True
+                out["genericWorker"]["config"]["d2gConfig"]["enableD2G"] = True
```

Comment:
does `out["genericWorker"]["config"]["d2gConfig"]` need to be created before `out["genericWorker"]["config"]["d2gConfig"]["enableD2G"]` ?

Reply:
I think you're right here Pete. Good callout. I'll update